File: src/client/java/io/github/seggan/deathview/client/mixin/AbstractWidgetAccessor.java
package io.github.seggan.deathview.client.mixin;

import net.minecraft.client.gui.components.AbstractWidget;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(AbstractWidget.class)
public interface AbstractWidgetAccessor {

    @Accessor
    void setAlpha(float alpha);

    @Accessor
    float getAlpha();
}

File: src/client/kotlin/io/github/seggan/deathview/client/mixin/AbstractWidgetAccessor.kt
package io.github.seggan.deathview.client.mixin

import net.minecraft.client.gui.components.AbstractWidget
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(AbstractWidget::class)
interface AbstractWidgetAccessor {

    @get:Accessor
    @set:Accessor
    var alpha: Float
}